profiles/schema.py

```diff
@@ -0,0 +1,67 @@
+import graphene
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+from munigeo.models import AdministrativeDivision
+from thesaurus.models import Concept
+
+from .models import DivisionOfInterest, Profile
+
+
+class ConceptType(DjangoObjectType):
+    class Meta:
+        model = Concept
+
+
+class AdministrativeDivisionType(DjangoObjectType):
+    class Meta:
+        model = AdministrativeDivision
+
+
+class DivisionOfInterestType(DjangoObjectType):
+    class Meta:
+        model = DivisionOfInterest
+
+
+class ProfileType(DjangoObjectType):
+    class Meta:
+        model = Profile
+
+    concepts_of_interest = graphene.List(ConceptType)
+    divisions_of_interest = graphene.List(AdministrativeDivisionType)
+
+
+class ProfileInput(graphene.InputObjectType):
+    nickname = graphene.String()
+    image = graphene.String()
+    email = graphene.String()
+    phone = graphene.String()
+    language = graphene.String()
+    contact_method = graphene.String()
+
+
+class CreateProfile(graphene.Mutation):
+    class Arguments:
+        profile = ProfileInput(required=True)
+
+    ok = graphene.Boolean()
+    profile = graphene.Field(ProfileType)
+
+    @login_required
+    def mutate(self, info, **kwargs):
+        profile_data = kwargs.pop("profile")
+        profile_data["user_id"] = info.context.user.id
+        profile = Profile.objects.create(**profile_data)
+        return CreateProfile(profile=profile)
+
+
+class Query(graphene.ObjectType):
+    profile = graphene.Field(ProfileType)
+    divisions_of_interest = graphene.List(DivisionOfInterestType)
```

Comment:
I would use `AdministrativeDivisionType `, since `DivisionOfInterest` has only internal logic value.
```python
class Query(graphene.ObjectType):
    ...
    divisions_of_interest = graphene.List(AdministrativeDivisionType)

    def resolve_divisions_of_interest(self, info, **kwargs):
        return AdministrativeDivision.objects.filter(division_of_interest__is_null=True)
```